src/components/daily/bolustooltip/BolusTooltip.js

```diff
@@ -172,10 +173,48 @@ class BolusTooltip extends React.Component {
     return extendedLine;
   }
 
+  getFormatedInputTime(bolus, timePrefs) {
+    const time = formatLocalizedFromUTC(bolus.inputTime, timePrefs, getHourMinuteFormat());
+    return time;
+  }
+
+  getBolusTypeLine(bolusType) {
+    return bolusType && (
+      <div className={styles.bolus}>
+        <div className={styles.label}>{t('Bolus')}</div>
+        <div className={styles.value}>{t(bolusType)}</div>
+      </div>
+      );
+  }
+
+  getIobLine(iob) {
+    return !!iob && (
+      <div className={styles.iob}>
+        <div className={styles.label}>{t('IOB')}</div>
+        <div className={styles.value}>{`${formatInsulin(iob)}`}</div>
+        <div className={styles.units}>U</div>
+      </div>
+    );
+  }
+
+  getPrescriptorLine(prescriptor) {
+    return prescriptor && prescriptor !== 'manual' && (      
+      <div className={styles.prescriptor}>
+        <div className={styles.label}>{t('Prescribed by Loop Mode')}</div>
+      </div>
+      );
+  }
+
   renderWizard() {
-    const wizard = this.props.bolus;
+    const { bolus, timePrefs } = this.props;
+    // to be renamed
```

Comment:
```suggestion
    // TODO: Rename Type to "calculator"
```
Same as platform ?
https://github.com/mdblp/platform/blob/889e7879b8e0d00d503d696c1addf8bf84962253/data/types/calculator/calculator.go#L18